pkgs/development/python-modules/ibis-framework/default.nix

```diff
@@ -68,65 +71,48 @@ buildPythonPackage rec {
     cached-property
     clickhouse-driver
     dask
+    datafusion
+    duckdb
+    duckdb-engine
     graphviz
+    importlib-metadata
     multipledispatch
     numpy
     pandas
     parsy
+    poetry-dynamic-versioning
     pyarrow
+    pydantic
     pytz
     regex
     requests
     sqlalchemy
-    tables
+    tabulate
     toolz
-  ] ++ lib.optionals (pythonOlder "3.8" && lib.versionOlder version "3.0.0") [
-    importlib-metadata
   ];
 
   checkInputs = [
     pytestCheckHook
     click
-    pytest
+    pytest-benchmark
```

Comment:
Can we easily remove pytest-benchmark?

Reply:
No, because the upstream package has default flags related to `pytest-benchmark`.